flaml/automl/automl.py

```diff
@@ -2358,6 +2401,10 @@ def custom_metric(
                 datasets, but will incur more overhead in time.
                 If dict: the dict contains the keywords arguments to be passed to
                 [ray.tune.run](https://docs.ray.io/en/latest/tune/api_docs/execution.html).
+            use_spark: boolean, default=False | Whether to use spark to run the training
+                in parallel spark jobs. This can be used to accelerate training on large models
+                and large datasets, but will incur more overhead in time and thus slow down
+                training in some cases.
```

Comment:
Shall we mention that gpu is not supported when use_spark=True?

Reply:
It's supported I think. Need more tests though.

Reply:
There is an argument "gpu_per_trial" which is related to this. If gpu is supported, lines involving it would need to be checked and adapted.

Reply:
resources_per_trial is not used in Sequential and Spark for now, but Spark cluster supports GPU, so I believe it can be supported in flaml[spark].

I added gpu is not supported when use_spark=True since we're now focusing on CPU clusters.